### DISCUSSION TITLE
Is there a rough ETA for a V3 release?

### DISCUSSION THREAD

**Discussion by OtherBarry** [roxiness/routify#528] [Q&A]
Just getting started with the Routify (and svelte) for the first time on a new project. There's a few features in V3 that I'd like to use eventually, but don't need yet.

If V3 is planning on releasing soon (say the next month or so) then I'll stick with V2 until it releases, but if it's going to be a while, then I'd rather start with the V3 beta now rather than having to port a lot of code between versions.

Totally understand that this is a community open source project, so not expecting any firm commitments or anything, just a general idea on progress.

**Comment by jakobrosenberg** [comment]
V3 is out. 🙂
